File-Formats/PDF/Working-with-OCR/Docker.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Performing OCR in Docker | Syncfusion
+description: Learn how to Perform OCR in Docker with easy steps using Syncfusion .NET OCR library.
+platform: file-formats
+control: PDF
+documentation: UG
+keywords: Assemblies
+---
+
+# How to Perform OCR on a PDF Document Using Docker container
+
+Optical Character Recognition (OCR) is a technology that converts scanned paper documents from PDF files or images into searchable and editable data.
+
+The [Syncfusion OCR processor library](https://www.syncfusion.com/document-processing/pdf-framework/net/pdf-library/ocr-process) has extended support to OCR process PDF documents and other scanned images in [docker](https://www.docker.com/why-docker/) container with the help of the [Tesseract](https://github.com/tesseract-ocr/tesseract) OCR engine.
+
+## Steps to perform OCR on a PDF document programmatically
+
+Step 1: Create a new ASP.NET Core application project.
+<img src="OCR-Images/OCRDocker1.png" alt="OCR Docker Step1" width="100%" Height="Auto"/>
+
+Step 2: In the project configuration window, name your project and select Next.
+<img src="OCR-Images/OCRDocker2.png" alt="OCR Docker Step2" width="100%" Height="Auto"/>
+
+Step 3: Enable the Docker support with Linux as a target OS.
+<img src="OCR-Images/OCRDocker3.png" alt="OCR Docker Step3" width="100%" Height="Auto"/>
+
+Step 4: Install the [Syncfusion.PDF.OCR.Net](https://www.nuget.org/packages/Syncfusion.PDF.OCR.Net/) NuGet package as a reference to your .NET Core application [NuGet.org](https://www.nuget.org/).
+<img src="OCR-Images/OCRDocker4.png" alt="OCR Docker Step4" width="100%" Height="Auto"/>
+
+Step 5: Tesseract assemblies are not added as a reference. They must be kept in the local machine, and the location of the assemblies is passed as a parameter to the OCR processor.
+{% highlight c# tabtitle="C#" %}
+
+OCRProcessor processor = new OCRProcessor(@"TesseractBinaries\")
+
+{% endhighlight %}
+Step 6: Place the Tesseract language data {E.g eng.traineddata} in the local system and provide a path to the OCR processor. Please use the OCR language data for other languages using the following [link](https://github.com/tesseract-ocr/tessdata)
```

Comment:
Please use the OCR language data for other languages using the following link,
[Tesseract language data](https://github.com/tesseract-ocr/tessdata) -

Reply:
changed the content